### DISCUSSION TITLE
787

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#959] [Comments]
# 787

"주말이나 여가시간에 개발하고 공부해야만 흔히 말하는 진짜 개발자인것 같은데, 본인은 그렇지 않는 것 같아 개발자를 계속해도 될까?" 같은 고민을 종종 듣게 된다.예전에 비슷한 고민을 한 적이 있어, 공감이 되는 고민이였다.친동생은 고등학생때 판타지 소설 단편을 써서 반 친구들한테 돌려보는 것에 엄청 재미를 느끼는 사람이였고, 그걸 계기로 문예창작학과로 진학을 했다.근데 막상 전문적으로 작가를 위한 학과를 가니 모든 교수님들이 작가란 순수문학을 해야하고, 판타지소설같은 대중문학은 작가로서는 지향해선 안된다라는 이야기를 많이 들었다.당시는 2008년~10년이다보니 아무래도 지금처럼 웹 소설과 같이 판타지, 로맨스 등의 소설 분야가 엄청난 인기를 끌던 때가 아니다보니 유독 더 심했던 것 같다.어찌됐든 그 분..

https://jojoldu.tistory.com/787

**Comment by jellyyelly** [comment]
자기만의 기준을 세우는게 진짜 중요하다고 느낍니다
남들이 만든 틀에 자신을 끼워 맞추는 순간 자신을 잃어버리는 느낌이에요

**Comment by mystyle2006** [comment]
개발자 뿐만 아니라 모든 영역에서 자신만의 기준을 시행착오를 겪으며 잡아가는 것이 정말 중요한 것 같아요. 중심을 잡을 수 있게되면 흔들리더라도 다시 중심을 잡는 방법을 배우게 되니깐요!

**Comment by Chadolbaegi128** [comment]
인생을 살아가는 데 있어서 나답게 사는 거만큼 중요한 게 없는 거 같습니다. 결국 내 인생의 주인은 자기 자신이니까요

**Comment by el-yun** [comment]
좋은 개발자가 되야하나 고민도 했지만 전 그냥 개발과 서비스를 만드는 게 즐거운데 고민이 필요할까싶더라구요. 뭔가 만들고 싶으면 집에서 혼자 만들기도 하고 궁금하면 찾아보고 성능이 안나오면 고쳐도보고 그게 공부아닐까 싶습니다.

**Comment by binary-ho** [comment]
많은 도움을 받았습니다. 좋은 글 감사합니다

**Comment by mag123c** [comment]
"내가 잘하고 있을까?"라는 회의감이 가끔 드는 요즘, 다시 초심을 되돌아보게 하는 글이네요. 빠르게 성장하고 싶었기 때문에 조급해지지 않았나 생각이 듭니다. 재밌어하는 일에 몰두하고, 재밌기 때문에 더 잘하고 싶어서 학습하고 공유하며 성장하는 긍정적인 사이클을 반복하다 보면 자연스레 성장할 것이라 믿습니다. 계속 가던 길을 가보겠습니다. 감사합니다!

**Comment by pine14** [comment]
좋은 글 감사합니다.

**Comment by mytory** [comment]
좋은 글 잘 읽었습니다. 동생분 이야기가 안타깝네요.

**Comment by KimCookieYa** [comment]
개추

**Comment by yonghyeokrhee** [comment]
서른 다섯에 개발을 시작하여 10대에 출발한 찐(?) 개발자에 뒤쳐진다 생각하며 늘 불안 했었습니다.
그런데 올해가 되어 드디어 향로님의 위 글이 무슨 말인지 이해가 되는 것 같아요. 오래 가는 것이 중요하고, 내가 생각하는 방식대로 하는 것이 중요하다는 것 말이죠.

**Comment by enKODING1** [comment]
"나 스스로 그 업을 더 흥미롭게 할 수 있는 나만의 업의 기준을 세우는 것" 이라는 말이 정말 마음에 와닿습니다. 

제가 생각해도 실무에서는 크게 도움이 되지 않는 프로젝트를 만드는 것을 좋아해요. 몇몇 분들은 실무에 도움되지 않는 것을 왜 만드냐 라는 말씀도 하시더라고요. 그 분들이 말하는 "개발자란 어떤 형태어야 한다" 라는 모습에 저를 끼우려 하다보니 저의 색깔을 잃어가는 느낌이 들었었어요.

그 분들에게서 한발짝 물러나 되 새겨보니, 제 자신에게 피드백 해야 할 부분도 정말 많더라고요. 타인의 틀에 나를 끼우기 보다 나에게 없는 타인의 특별한 시선 중 내게 부족한 것들을 찾아 채워가야겠구나 하게 되었어요.

이 글을 읽으며 누군가에게 실무에 도움되지 않아 보이는 프로젝트가 "개발을 계속 해 나갈 수 있는 나만의 방법"을 찾아가고 있었던 거였구나 하는 생각이 들었습니다.

이제는 "나만의 방법" + "타인의 특별한 시선으로 얻은 부족한 점"을 잘 새기며 좋은 개발자로 성장해 나가도록 해야겠습니다. 좋은 글 감사합니다!
